android/app/src/main/AndroidManifest.xml

```diff
@@ -1,5 +1,10 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.mendix.nativetemplate">
+          xmlns:tools="http://schemas.android.com/tools"
+          package="com.mendix.nativetemplate">
+
+    <!--    Remove the next line after update react-native-inappbrowser-->
+    <uses-permission android:name="android.permission.QUERY_ALL_PACKAGES" tools:node="remove"
```

Comment:
Is it backwards compatible? I see that `android.permission.QUERY_ALL_PACKAGES` was introduced in Android R.

Reply:
In that case, we remove this permission from manifest by attribute`tools:node="remove`

Reply:
Perfect